app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFieldsRepository.scala

```diff
@@ -36,9 +37,14 @@ import scala.concurrent.Future
 @ImplementedBy(classOf[SubscriptionFieldsMongoRepository])
 trait SubscriptionFieldsRepository {
 
-  //TODO consider Future[Boolean]
-  def save(subscription: SubscriptionFields): Future[Unit]
+  /**
+    * Saves or inserts entity depending on if it already exists.
```

Comment:
Do not worry about scaladoc. 
Let's remove it from anywhere.
If the code does something not easy to read or you need to add some notes, you can add a comment above the instruction.

Reply:
done